java/code/src/com/redhat/rhn/common/util/http/HttpClientAdapter.java

```diff
@@ -101,9 +103,12 @@ public HttpClientAdapter() {
         Optional<SSLConnectionSocketFactory> sslSocketFactory = Optional.empty();
         try {
             SSLContext sslContext = SSLContext.getDefault();
+            List<String> supportedProtocols = Arrays.asList(sslContext.getSupportedSSLParameters().getProtocols());
+            List<String> wantedProtocols = Arrays.asList("TLSv1", "TLSv1.1", "TLSv1.2", "TLSv1.3");
```

Comment:
TLSv1.1 is for SLE11 systems, right? But why TLSv1?

Reply:
SLE11 can only 1.0, even 1.1 is not supported by it.

Reply:
Ah, right. Sorry for the noise.